### DISCUSSION TITLE
Poor quality when using multilingual vocoder for persian language

### DISCUSSION THREAD

**Discussion by arminsoltan** [coqui-ai/TTS#1468] [General Q&A]
Hi, I used tacotron 2 for training persian dataset (around 33 hours). When I use Griffin-Lim, quality of sound is good but when I used other pretrained coqui vocoder such as fullband-melgan and wavegrad, sound quality is poor.
Here are my config file and two sound file that use two distinct vocoder. Below command is for generating soundfile with fullband melgan vocoder.
```
tts --text "سلام حال شما خوب است؟" --model_path "best_model.pth" --config_path "config.json" --vocoder_name "vocoder_models/universal/libri-tts/fullband-melgan" --out_path "example.wav"
```

[config.zip](https://github.com/coqui-ai/TTS/files/8404687/config.zip)
[sound_file_by_full_band_melgan.zip](https://github.com/coqui-ai/TTS/files/8404689/sound_file_by_full_band_melgan.zip)
[sound_file_by_grffin_lim.zip](https://github.com/coqui-ai/TTS/files/8406962/sound_file_by_grffin_lim.zip)

I will appreciate in advance if you can help me.

**Comment by lexkoro** [comment]
Most likely the audio parameters between vocoder and tts config don't match.
The vocoder and the tts model should have the same audio parameters in the config.

**Reply by pooriaarab** [reply]
@arminsoltan heyy i want to clone a persian voice. is there any guide on how to do it using this repo?
